Clock/Clock.qml

```diff
@@ -26,4 +31,9 @@ Item {
             }
         }
     }
+
+    Timer { //l'ora si aggiorna ogni 0.5 secondi
+        interval: 500; running: true; repeat: true
+        onTriggered: { currentDate=new Date() }
+    }
```

Comment:
Messo qui questo timer rende obsoleti un po' di timer in giro e un po' di codice in giro. Non sono del tutto sicuro che hai pulito tutto, puoi per favore ricontrollare per esserne sicuro?